ci-operator/config/redhat-developer/devspaces-interop-tests/redhat-developer-devspaces-interop-tests-main__devspaces-ocp4.15-lp-interop.yaml

```diff
@@ -41,11 +41,11 @@ tests:
       FIREWATCH_CONFIG: |
         [
           {"step": "ipi-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp"]},
-          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPINTEROP", "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp"]},
+          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPINTEROP", "group": {"name": "operator", "priority": 1}, "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp"]},
           {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp"]},
           {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
           {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPINTEROP", "group": {"name": "cluster", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp"]},
-          {"step": "devspaces-tests", "failure_type": "all", "classification": "PQE test failure", "jira_project": "CRW", "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp","interop-tests"]}
+          {"step": "devspaces-tests", "failure_type": "all", "classification": "PQE test failure", "jira_project": "CRW", "group": {"name": "operator", "priority": 2}, "jira_additional_labels": ["4.15-lp","self-managed-lp","devspaces-lp","interop-tests"]}
```

Comment:
The priority here makes sense I'm just questioning the use of the operator group in this line which is running the layered product tests. Based on the definition in the description you provided in this PR. This step would fall under the lp-tests group, but there would only be 1 step so that wouldn't require a priority.

It seems to me in this case that choosing either operator or lp-tests would make sense, like you've done here, I just want to be sure we are being consistent.

Thinking outloud here. Why not just use strictly the lp-tests group instead of a mix of lp-tests and operator in this PR.

- If we use just lp-tests then the priority 1 for that group will always be reserved for the installation of the operator, then priority 2 would be for setup steps or test execution if there are not other setup steps.